README.md

```diff
@@ -370,6 +402,39 @@ let () =
   painIndexMap |> Hashtbl.iter (fun k v -> Js.log {j|key:$k, val:$v|j})
 ```
 
+###### Belt.HashMap
+
+Use a generic `Belt.HashMap` to demo a mutable hash map with string type key. In fact, You can use `Belt.HashMap.String` instead.
+
+```ml
+open Belt
+module StrHash =
+  Id.MakeHashable(struct
+                    type t = string
+                    let hash (x : t) = String.length x
+                    let eq (a : t) (b : t) = a = b
+                  end)
+
+let painIndexMap = HashMap.make ~hintSize:10 ~id:(module StrHash)
+
+let () =
+  let open HashMap in
+
+    set painIndexMap "western paper wasp" 1.0;
+    set painIndexMap "yellowjacket" 2.0;
+    set painIndexMap "honey bee" 2.0;
+    set painIndexMap "red paper wasp" 3.0;
+    set painIndexMap "tarantula hawk" 4.0;
+    set painIndexMap "bullet ant" 4.0
+
+let () =
+  HashMap.set painIndexMap "bumble bee" 2.0
+
+let () =
+  HashMap.forEach painIndexMap
+    (fun k  -> fun v  -> Js.log {j|key:$k, val:$v|j})
```

Comment:
The explicit currying and double whitespace formatting is a bit weird. I assume it's because it's auto-translated from Reason?